swtbot_test/src/testsuites/RemoveAzureModuleAndGenerateCode.java

```diff
@@ -0,0 +1,228 @@
+		RTOSManager.loadRTOSModel(new File(Utility.getBundlePath(LogUtil.PLUGIN_ID, RTOS_PG_XML_FILE)));
+		projectModelSpecific = PGUtility.prepareProjectModel(RTOSType.AZURE, RTOSVersion.Azure_6_2_1, RTOSApplication.AZURE_BARE, Constants.CCRX_TOOLCHAIN, TargetBoard.BOARD_RSK_RX65N_2MB);
+	}
+	
+	@Test
+	public void tc_01_CreateThreadxProject() throws Exception{
+		PGUtility.createProject(RTOSType.AZURE, RTOSVersion.Azure_6_2_1, RTOSApplication.AZURE_BARE, Constants.CCRX_TOOLCHAIN, TargetBoard.BOARD_RSK_RX65N_2MB);
+		
+	}
+	
+	@Test
+	public void tc_02_AddComponentFilex() throws Exception{
+		Utility.openSCFGEditor(projectModelSpecific);
+		Utility.addComponentAndGenerate("filex");
+		Utility.getProjectExplorerView().setFocus();
+		SWTBotTreeItem project= bot.tree().getTreeItem(projectModelSpecific.getProjectName() + " ["+ projectModelSpecific.getActiveBuildConfiguration() +"]");
+		project.getNode(ProjectParameters.FolderAndFile.FOLDER_LIBS).expand();
+		project.getNode(ProjectParameters.FolderAndFile.FOLDER_LIBS).getNode(ProjectParameters.RTOSComponent.FILEX).expand();
+		SWTBotTreeItem[] items = project.getNode(ProjectParameters.FolderAndFile.FOLDER_LIBS).getNode(ProjectParameters.RTOSComponent.FILEX).getItems();
+		boolean isFileExist = false;
+		for(SWTBotTreeItem item: items) {
+			if(item.getText().contains(ProjectParameters.FolderAndFile.FILE_FX_USER_H)) {
+				isFileExist = true;
+			}
+		}
+		if (!isFileExist) {
+			assertFalse(true);
+		}
+	}
+	@Test
+	public void tc_03_AddComponentNetxduo() throws Exception{
+		bot.editorByTitle(projectModelSpecific.getProjectName() + ".scfg").setFocus();
+		Utility.addComponentAndGenerate("netx");
+		Utility.getProjectExplorerView().setFocus();
+		SWTBotTreeItem project = bot.tree().getTreeItem(projectModelSpecific.getProjectName() + " ["+ projectModelSpecific.getActiveBuildConfiguration() +"]");
+		project.getNode(ProjectParameters.FolderAndFile.FOLDER_LIBS).expand();
+		project.getNode(ProjectParameters.FolderAndFile.FOLDER_LIBS).getNode(ProjectParameters.RTOSComponent.NETXDUO)
+				.expand();
+		SWTBotTreeItem[] items = project.getNode(ProjectParameters.FolderAndFile.FOLDER_LIBS)
+				.getNode(ProjectParameters.RTOSComponent.NETXDUO).getItems();
+		boolean isFileExist = false;
+		for (SWTBotTreeItem item : items) {
+			if (item.getText().contains(ProjectParameters.FolderAndFile.FILE_NX_USER_H)) {
+				isFileExist = true;
+			}
+		}
+		if (!isFileExist) {
+			assertFalse(true);
+		}
+	}
+	@Test
+	public void tc_04_AddComponentNetxduoAddons() throws Exception{
+		bot.editorByTitle(projectModelSpecific.getProjectName() + ".scfg").setFocus();
+		Utility.addComponentAndGenerate("netx duo addons");
+		Utility.getProjectExplorerView().setFocus();
+		SWTBotTreeItem project = bot.tree().getTreeItem(projectModelSpecific.getProjectName() + " ["+ projectModelSpecific.getActiveBuildConfiguration() +"]");
+		project.getNode(ProjectParameters.FolderAndFile.FOLDER_LIBS).expand();
```

Comment:
E: too complicated, no need to migrate 100% from rcptt, check for component added in component tree is enough

Reply:
Fixed in latest commit